src/ui/settings/ViewAppPermissions.qml

```diff
@@ -0,0 +1,158 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.5
+import QtQuick.Controls 2.15
+import QtGraphicalEffects 1.15
+import QtQuick.Layouts 1.15
+import Mozilla.VPN 1.0
+import "../components"
+import "../themes/themes.js" as Theme
+
+VPNFlickable {
+    id: vpnFlickable
+    readonly property int defaultMargin: 18
+     property bool vpnIsOff: (VPNController.state === VPNController.StateOff)
+
+    flickContentHeight: menu.height+enableAppList.height+applistLabel.height+applist.contentHeight+vpnOnAlert.height+defaultMargin+Theme.windowMargin+applistLabel.anchors.topMargin+applist.anchors.topMargin
+
+    Component.onCompleted: {
+       VPNAppPermissions.requestApplist();
+    }
+
+    VPNMenu {
+        id: menu
+        title: qsTrId("vpn.settings.appPermissions")
+        isSettingsView: true
+    }
+
+    VPNCheckBoxRow {
+        id: enableAppList
+
+        anchors.top: menu.bottom
+        anchors.topMargin: Theme.windowMargin
+        width: parent.width - Theme.windowMargin
+
+        //% "Protect specific apps"
+        labelText: qsTrId("vpn.settings.protectSelectedApps")
+        //% "Choose which apps should use the VPN"
+        subLabelText: qsTrId("vpn.settings.protectSelectedApps.description")
+        isChecked: (VPNSettings.protectSelectedApps)
+        isEnabled: vpnFlickable.vpnIsOff
+        showDivider: false
+        onClicked: VPNSettings.protectSelectedApps = !VPNSettings.protectSelectedApps
+
+    }
+
+    VPNCheckBoxAlert {
+        id: vpnOnAlert
+        anchors.top: enableAppList.bottom
+        visible: !vpnFlickable.vpnIsOff
+    }
+
+
+
+    VPNBoldLabel {
+        id: applistLabel
+        visible: VPNSettings.protectSelectedApps
+
+        anchors.top: vpnFlickable.vpnIsOff? enableAppList.bottom : vpnOnAlert.bottom
+        anchors.topMargin: 28
+        anchors.left: parent.left
+        anchors.leftMargin: Theme.windowMargin
+        width: parent.width
+        //% "Installed Applications"
+        //: Header for the installed Applications
+        text: qsTrId("vpn.settings.installedApps")
+        Accessible.role: Accessible.Heading
+    }
+
+
+    VPNList {
+        id: applist
+        visible: VPNSettings.protectSelectedApps && count > 0
+
+        anchors.top: applistLabel.bottom
+        anchors.topMargin: 16
+        anchors.left: parent.left
```

Comment:
Let's change this to `vpnFlickable.left` so that all of the check boxes are uniformly aligned.